recipes/mikktspace/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "cci.20200325":
+    url: "https://github.com/mmikk/MikkTSpace/archive/master.zip"
+    sha256: "c4377c330d929134b4990567c329302f9e00eb818fde4ff31967e206926536ff"
```

```suggestion
    url: "https://github.com/mmikk/MikkTSpace/archive/3e895b49d05ea07e4c2133156cfa94369e19e409.tar.gz"
    sha256: "aeba65ddee85a679133d510d71d00b108f603752f90b15b9ecf7777033f6e351"
```

Use a specific commit, not the branch name: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases